File: app/src/main/java/com/rdev/tryp/ui_only/screens/invite2/Invite2Fragment.java
package com.rdev.tryp.ui_only.screens.invite2;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.rdev.tryp.R;

import java.util.ArrayList;
import java.util.List;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

/**
 * Created by Alexey Matrosov on 02.03.2019.
 */
public class Invite2Fragment extends Fragment {
    private List<InviteItem> hereItems;
    private List<InviteItem> inviteItems;

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_invite2, container, false);

        fillFakeData();

        RecyclerView hereRecyclerView = view.findViewById(R.id.invite_2_here_container);
        hereRecyclerView.setAdapter(new InviteFriendsAdapter(getContext(), hereItems));
        hereRecyclerView.setLayoutManager(new LinearLayoutManager(getContext()));

        RecyclerView inviteRecyclerView = view.findViewById(R.id.invite_2_invite_container);
        inviteRecyclerView.setAdapter(new InviteFriendsAdapter(getContext(), inviteItems));
        inviteRecyclerView.setLayoutManager(new LinearLayoutManager(getContext()));
        return view;
    }

    void fillFakeData() {
        hereItems = new ArrayList<>();
        hereItems.add(new InviteItem(R.drawable.test_friend_icon, "Alex Bingo", "+1-200-4000-2525", InviteItem.Status.Invited));
        hereItems.add(new InviteItem(0, "Alex Bingo", "+1-200-4000-2525", InviteItem.Status.Followed));
        hereItems.add(new InviteItem(0, "Alex Bingo", "+1-200-4000-2525", InviteItem.Status.Invited));

        inviteItems = new ArrayList<>();
        inviteItems.add(new InviteItem(0, "Alex Bingo", "+1-200-4000-2525", InviteItem.Status.NotInvited));
        inviteItems.add(new InviteItem(0, "Alex Bingo", "+1-200-4000-2525", InviteItem.Status.NotInvited));
        inviteItems.add(new InviteItem(0, "Alex Bingo", "+1-200-4000-2525", InviteItem.Status.NotInvited));
    }
}

File: app/src/main/java/com/rdev/tryp/blocks/screens/invite2/Invite2Fragment.kt
package com.rdev.tryp.blocks.screens.invite2

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.rdev.tryp.R

import java.util.ArrayList
import androidx.fragment.app.Fragment
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView

/**
 * Created by Alexey Matrosov on 02.03.2019.
 */


class Invite2Fragment : Fragment() {

    private var hereItems: MutableList<InviteItem>? = null
    private var inviteItems: MutableList<InviteItem>? = null

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.fragment_invite2, container, false)

        fillFakeData()

        val hereRecyclerView = view.findViewById<RecyclerView>(R.id.invite_2_here_container)
        hereItems?.let { items -> hereRecyclerView.adapter = InviteFriendsAdapter(items) }
        hereRecyclerView.layoutManager = LinearLayoutManager(context)

        val inviteRecyclerView = view.findViewById<RecyclerView>(R.id.invite_2_invite_container)
        inviteItems?.let { items -> inviteRecyclerView.adapter = InviteFriendsAdapter(items) }
        inviteRecyclerView.layoutManager = LinearLayoutManager(context)
        return view
    }

    private fun fillFakeData() {
        hereItems = ArrayList()
        hereItems?.add(InviteItem(R.drawable.test_friend_icon, "Alex Bingo", "+1-200-4000-2525", InviteItem.Status.Invited))
        hereItems?.add(InviteItem(0, "Alex Bingo", "+1-200-4000-2525", InviteItem.Status.Followed))
        hereItems?.add(InviteItem(0, "Alex Bingo", "+1-200-4000-2525", InviteItem.Status.Invited))

        inviteItems = ArrayList()
        inviteItems?.add(InviteItem(0, "Alex Bingo", "+1-200-4000-2525", InviteItem.Status.NotInvited))
        inviteItems?.add(InviteItem(0, "Alex Bingo", "+1-200-4000-2525", InviteItem.Status.NotInvited))
        inviteItems?.add(InviteItem(0, "Alex Bingo", "+1-200-4000-2525", InviteItem.Status.NotInvited))
    }

}